dspace_metadata_import.sh

```diff
@@ -3,31 +3,67 @@
 
 ########################################################################################
 ## Run this script to import metadata from old dspace into new dspace6 database, and  ##
-## copy over the metadata registry files.											  ##
+## copy over the metadata registry files.                                             ##
 ########################################################################################
 
 
 ## Refer to : https://unix.stackexchange.com/questions/191990/shell-script-to-execute-psql-command ##
 ## DSpace6 Upgrading: https://wiki.duraspace.org/display/DSDOC6x/Upgrading+DSpace ##
 
 
-#########################################################################
-#                       									 			#
-#	Need to 															#
-#		1) copy over the dumped pgsql file from old dspace, and			#
-#		2) set up the ownership, and									#
-#		3) replace the user to be dspace or something, and 				#
-#		4) comment out the block of adding rule protect_approval_upd	#
-#    	5) comment out the privileges granted to user libacct			#
-#                       												#
-#########################################################################
-
-PATH_TO_REGISTRIES_FROM_OLD_DSPACE=/tmp/registries.tar.gz
-PATH_TO_OLD_DSPACE_DATABASE_DATA=/tmp/test3.pgsql
-EXISTING_DSPACE_DATABASE_NAME=dspace
-EXISTING_DSPACE_DATABASE_NAME_CHANGED=dspace_old
-NEW_DSPACE_DATABASE_NAME=dspace2
-DSPACE_DATABASE_OWNER=dspace
+#################################################################################
+#                                                                               #
+#       Need to                                                                 #
+#		1) copy over the dumped pgsql file from old dspace, and         #
+#		2) set up the ownership, and                                    #
+#		3) replace the user to be dspace or something, and              #
+#		4) comment out the block of adding rule protect_approval_upd    #
+#               5) comment out the privileges granted to user libacct           #
+#                                                                               #
+################################################################################
+
+
+EXISTING_DSPACE_DATABASE_NAME_CHANGED=dspace_changed_from_existing_temp
+NEW_DSPACE_DATABASE_NAME=dspace_new_temp
+
+while getopts o:p:q:r:s:t: option
+  do
+    case "${option}"
+    in
+      o) PATH_TO_REGISTRIES_FROM_OLD_DSPACE=${OPTARG};;
```

Comment:
Not a huge deal, but I'd either use positional arguments or make these command line switches more mnemonic. Just using `-p -q -r -s -t` seems a little odd.